js/views/chatVw.js

```diff
@@ -113,9 +130,13 @@ module.exports = baseVw.extend({
         filteredMd = this.filteredChatConvos.findWhere({ guid: md.get('guid') });
         filteredMd && filteredMd.set(md.attributes);
       }
-
-      md.hasChanged('unread') && this.setAggregateUnreadCount();          
-    });      
+      
+      if (md.hasChanged('unread') && md.get('unread') > 0) {
```

Comment:
I don't think we want to include the `> 0` check, because when a conversation's messages are marked as read, it will go from `x` (e.g. 12) to 0. In this case we will want to call `setAggregateUnreadCount` so the task bar counts (currently only visible on OSX) are updated. I imagine we'd want to re-sort as well.